js/views/reviewsVw.js

```diff
@@ -0,0 +1,103 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    RatingCl = require('../collections/ratingCl'),
+    baseVw = require('./baseVw'),
+    ReviewView = require('./reviewVw');
+
+module.exports = baseVw.extend({
+
+  className: "flexCol-12 flex-border js-tabTarg js-reviews hide",
+
+  events: {
+  },
+
+  VIEWS_PER_BATCH: 25,
+
+  MAX_VIEWS: 1500,
```

Comment:
Since we're fetching a result set of unknown size, I cap the max number of results we'll render to the value above. There is some limit as to how much the browser can reasonable render and still perform well. I have no idea what that is as I'm sure it varies depending on your set-up, so I took my best guess.

Should we want to display result sets larger than the limit, we'll need to implement pagination or maybe bi-directional lazy loading. I think the latter could be really cool.